Dockerfile

```diff
@@ -245,9 +245,21 @@ USER buildbot
 RUN /bin/bash -c 'phpbrew init && source ~/.phpbrew/bashrc && phpbrew install 5.6 +default && \
     phpbrew app get composer'
 
+################################################################################
+#
+# Emacs Cask
+#
+################################################################################
 USER root
+RUN add-apt-repository ppa:kelleyk/emacs
```

Comment:
I'm not a ubuntu user, I'm happy to hear any suggest on how to install emacs 25 on ubuntu officially.

AFAICT ubuntu 14 officially only provide emacs 24.3, but most moden emacs packages require at lease 24.4.

CMIIW i think kelleyk/emacs just provide deb pacakge, it's still build from official emacs source